src/main/resources/application.properties

```diff
@@ -7,13 +7,17 @@ uk.gov.notify.email.templates=${UK_GOV_NOTIFY_EMAIL_TEMPLATES:\
   FR_CONSENT_ORDER_MADE:'16688985-341d-4628-9406-46d9a8007f53',\
   FR_CONSENT_ORDER_NOT_APPROVED: '474f0be7-6dd8-4f1c-81de-da7973d27ac0',\
   FR_CONSENT_ORDER_AVAILABLE: '9d2c1fe4-167d-4b34-af42-993d0bc72bc7',\
-  FR_CONTESTED_HWF_SUCCESSFUL: '86d66fb5-95a1-41cc-9956-c730fa81d35e'}}
-uk.gov.notify.email.template.vars={FR_HWF_SUCCESSFUL: {frEmail:'HMCTSFinancialRemedy@justice.gov.uk', contactNumber:'0300 303 0642', linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/' },\
+  FR_CONTESTED_HWF_SUCCESSFUL: '86d66fb5-95a1-41cc-9956-c730fa81d35e',\
+  FR_CONTESTED_PREPARE_FOR_HEARING: 'f1ee98aa-c6fe-4226-a309-d75e4dd0f8dd'}}
+uk.gov.notify.email.template.vars={\
+  FR_HWF_SUCCESSFUL: {frEmail:'HMCTSFinancialRemedy@justice.gov.uk', contactNumber:'0300 303 0642', linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/' },\
   FR_ASSIGNED_TO_JUDGE: {frEmail:'HMCTSFinancialRemedy@justice.gov.uk', contactNumber:'0300 303 0642', linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/' },\
   FR_CONSENT_ORDER_MADE: {frEmail:'HMCTSFinancialRemedy@justice.gov.uk', contactNumber:'0300 303 0642', linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/' },\
   FR_CONSENT_ORDER_NOT_APPROVED: {frEmail:'HMCTSFinancialRemedy@justice.gov.uk', contactNumber:'0300 303 0642', linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/'},\
   FR_CONSENT_ORDER_AVAILABLE: {frEmail:'HMCTSFinancialRemedy@justice.gov.uk', contactNumber:'0300 303 0642', linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/'},\
-  FR_CONTESTED_HWF_SUCCESSFUL: {frEmail:'', contactNumber:'',linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/'}}
+  FR_CONTESTED_HWF_SUCCESSFUL: {frEmail:'', contactNumber:'',linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/'},\
+  FR_CONTESTED_PREPARE_FOR_HEARING: {frEmail:'HMCTSFinancialRemedy@justice.gov.uk', contactNumber:'0300 303 0642',linkToSmartSurvey:'http://www.smartsurvey.co.uk/s/KCECE/'}\
+}
```

Comment:
May @shaunganley and I get feedback on whether this information is correct - @johnoneill46 / @yarpo

Reply:
I'm afraid I don't know.

Reply:
Where did you get it from?

Reply:
I copied the data from above in the file. But even that is inconsistent